saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/servlet/filter/Saml2WebSsoAuthenticationRequestFilter.java

```diff
@@ -121,6 +124,16 @@ public void setRedirectMatcher(RequestMatcher redirectMatcher) {
 		this.redirectMatcher = redirectMatcher;
 	}
 
+	/**
+	 * Use the given {@link Saml2AuthenticationRequestContextResolver} that creates a {@link Saml2AuthenticationRequestContext}
+	 *
+	 * @param authenticationRequestContextResolver the {@link Saml2AuthenticationRequestContextResolver} to use
+	 */
```

Comment:
Will you please add a `@since 5.4` since that this the release where this will get added?

Reply:
Done.